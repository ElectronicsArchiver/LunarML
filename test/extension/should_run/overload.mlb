$(SML_LIB)/basis/basis.mlb
ann "allowOverload true" in
  overload.sml
end
