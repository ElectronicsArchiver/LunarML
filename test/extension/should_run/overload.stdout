I
O
I
O
O
true
false
